assets/src/edit-story/components/library/tabs/index.js

```diff
@@ -37,75 +45,170 @@ const Tabs = styled.ul`
 `;
 
 const Tab = styled.li`
-  width: 64px;
+  width: 72px;
   height: 100%;
   color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
-const Icon = styled.a`
+const QuickAction = styled.button`
+  margin: 0 0 0 -10px;
+  background: transparent;
+  border: 0;
+  padding: 0;
+  line-height: 1;
+  position: relative;
+  color: ${({ theme }) => theme.colors.fg.v1};
+
+  &:hover::before {
+    content: '';
+    display: block;
+    position: absolute;
+    background: rgba(255, 255, 255, 0.3);
+    left: 5px;
+    right: 5px;
+    top: 5px;
+    bottom: 5px;
+  }
+`;
+
+const IconContent = styled.div`
+  width: 60%;
+  height: 80%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const IconButton = styled.button.attrs(({ isActive }) => ({
+  tabIndex: isActive ? 0 : -1,
+}))`
   color: inherit;
   background: ${({ isActive, theme }) =>
     isActive ? theme.colors.bg.v4 : 'transparent'};
+  border: 0;
+  padding: 0;
+  margin: 0;
+  width: 100%;
   height: 100%;
   display: flex;
   justify-content: center;
   align-items: center;
 
-  &:hover {
-    color: inherit;
+  &:focus {
+    outline: none;
+    & ${IconContent} {
+      /* todo: how to show default outline on inner element cross-browser? */
+      outline: -webkit-focus-ring-color auto 5px;
+    }
   }
 
   ${({ isActive }) =>
     !isActive &&
     `
-  opacity: .4;
+  opacity: .5;
   &:hover { opacity: 1; }
   `}
 
   svg {
-    width: 22px;
-    height: 22px;
+    display: block;
+    width: 28px;
+    height: 28px;
   }
 `;
 
 function Media(props) {
   return (
     <Tab>
-      <Icon {...props}>
-        <MediaIcon />
-      </Icon>
+      <IconButton {...props}>
+        <IconContent>
+          <MediaIcon aria-label={__('Media library', 'web-stories')} />
+        </IconContent>
+      </IconButton>
     </Tab>
   );
 }
 
 function Text(props) {
+  const {
+    actions: { insertElement },
+  } = useLibrary();
+
+  const handleAddText = (evt) => {
+    evt.stopPropagation();
+    evt.preventDefault();
+
+    insertElement('text', {
+      content: 'Double-click to edit...',
```

Comment:
Would need to be translatable

Reply:
Ah yes, it's basically just dummy, but might survive all the way to product 😉. Will add i18n.